include/aspect/geometry_model/interface.h

```diff
@@ -346,6 +346,14 @@ namespace aspect
         void
         parse_parameters (ParameterHandler &prm);
 
+        /**
+         * Collects periodic boundaries constraints for the given geometry.
+         * Currently valid for box and 2D quarter shell geometries
```

Comment:
Please remove the second sentence from the comment. This is going to change, and we will forget to update this. Instead add a bit of information to what the input argument is ("@p X" is doxygen code for: link to the input parameter with the name X). E.g. like this:


```suggestion
         * Collects periodic boundaries constraints for the given geometry,
         * which will be added to the existing @p constraints.
```